src/components/Breadcrumbs.tsx

```diff
@@ -68,7 +68,7 @@ function Breadcrumbs({breadcrumbs, style}: BreadcrumbsProps) {
                     <Text style={[styles.breadcrumbSeparator]}>/</Text>
                     <Text
                         numberOfLines={1}
-                        style={[styles.mw75, styles.flexShrink0, styles.breadcrumb]}
+                        style={[styles.mw75, styles.flexShrink0, styles.breadcrumb, styles.flex1]}
```

Comment:
✋ Coming from https://github.com/Expensify/App/issues/44608

Adding the `flex1` causes the above issue. For the solution we conditionally apply the `flex1` only for the primary breadcrumb. For more detail in the [proposal](https://github.com/Expensify/App/issues/44608#issuecomment-2196796136).